addons/monitoring-standalone/v1.9.0.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: heapster
+  namespace: kube-system
+  labels:
+    k8s-addon: monitoring-standalone.addons.k8s.io
+    k8s-app: heapster
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+    version: v1.9.0
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: heapster
+      version: v1.9.0
+  template:
+    metadata:
+      labels:
+        k8s-app: heapster
+        version: v1.9.0
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      serviceAccountName: heapster
+      containers:
+        - image: k8s.gcr.io/heapster-amd64:v1.5.3
+          name: heapster
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8082
+              scheme: HTTP
+            initialDelaySeconds: 180
+            timeoutSeconds: 5
+          resources:
+            # keep request = limit to keep this container in guaranteed class
+            limits:
+              cpu: 100m
+              memory: 300Mi
+            requests:
+              cpu: 100m
+              memory: 300Mi
+          command:
+            - /heapster
+            - --source=kubernetes.summary_api:''
+        - image: k8s.gcr.io/addon-resizer:2.1
```

Comment:
What's the source for this?  I'm trying to make sense of the version of addon-resizer - 2.1 seems older than 1.8.1

Reply:
I can't find the source to this, what appeared to be, "newer" version. I will revert it back to the now 1.8.3